userguide/content/en/about/index.md

```diff
@@ -2,7 +2,6 @@
 title: About Docsy
 linkTitle: About
 menu: {main: {weight: 10}}
-layout: docs
```

Comment:
@LisaFC et al.: prior to deleting line 5 above, there was a `_print` version of the about page, which pretty much looked like the non-print version. With line 5 gone, we've lot the `_print` version of the About page. Is that ok with you? Given that the two pages are identical AFAICT.

Reply:
For the record, this is what the `_print` page looked like (excerpt), _before_ this PR was merged:

> <img width="930" alt="image" src="https://user-images.githubusercontent.com/4140793/224163060-75867ade-b0b3-46eb-98a5-5f7756212d8c.png">

Reply:
I don't think that's a problem, given that the page was exactly the same.